test/e2e/volumes.go

```diff
@@ -381,9 +381,8 @@ func deleteCinderVolume(name string) error {
 	return err
 }
 
-// These tests need privileged containers, which are disabled by default.  Run
-// the test with "go run hack/e2e.go ... --ginkgo.focus=[Feature:Volumes]"
-var _ = framework.KubeDescribe("Volumes [Feature:Volumes]", func() {
+// These tests need privileged containers, which are disabled by default.
```

Comment:
@msau42 I don't see that change but maybe I missed it. It seems like adding a priv bool in `VolumeTestConfig` would be a flexible way of allowing the caller to decide if the server-pod should be privileged. Can you point me to the pr? Thanks.